templates/etc/squid3/squid.conf.j2

```diff
@@ -2276,7 +2276,7 @@ http_port {{ squid_http_port }} transparent
 #			See ICP and HTCP options below for additional details.
 #
 #
-{% if squid_cache_peering %}
+{% if squid_cache_peering is defined %}
```

Comment:
The way this is now is if it is true then the settings are configured otherwise if it is false they are not. So I am not really sure what you are saying here. The if it is defined honestly makes no sense at this point because it is already defined by default as false.